workflows/data-manipulation/split-collection/README.md

```diff
@@ -0,0 +1,14 @@
+# Split collection
+
+These workflows allow to split a collection into 2 using identifiers.
+
+The common input to all workflows is a collection of type 'list'.
+
+The way to split the collection differs with the workflow.
+
+- In the workflow "Split collection by pattern in identifiers", you need to specify a "pattern". This is a word that is present only in one part of your samples. This will split your collection into 2: one with the identifiers which have the 'pattern' and the other one with the identifiers which don't have.
```

Comment:
Is that a regex pattern or literal words ?

Reply:
It uses grep so it is a regex. However, as this workflow is targeting a "user" audience I was trying to make it understandable for non bioinformaticians but I am happy to get suggestions.

Reply:
They might be surpised if they do enter special characters, so best to mention this I think ?

Reply:
Actually, do you expect users to run this workflow as a standalone workflow ? I thought this was useful as a thing to embed in another workflow ? Would it maybe make more sense to add what is not covered by the filter filter to the tool itself ?

Reply:
I feel like I am using this workflow as a standalone. For example, to select only part of my samples for deeper analysis or for a specific heatmap etc...

You are right about the word/regex, I should warn the users.